py/conftest.py

```
@@ -187,6 +196,14 @@ def get_options(driver_class, config):
             options.add_argument("--headless=new")
         if driver_class == "Firefox":
             options.add_argument("-headless")
+
+    if bidi:
+        if not options:
+            options = getattr(webdriver, f"{driver_class}Options")()
+
+        if driver_class == "Chrome" or driver_class == "Edge" or driver_class == "Firefox":
```

Do we need to specify this? If a dev tries to run bidi tests on ie it'll give the right failure message instead of ignoring the flag.

---

Should be right, I'll remove